app/models/campus_access.rb

```diff
@@ -0,0 +1,14 @@
+require 'csv'
+
+class CampusAccess < ActiveRecord::Base
+  def self.has_access?(uid)
+    where(uid: uid).count.positive?
+  end
```

Comment:
@carolyncole where will you use this?

Reply:
@christinach the third step in https://github.com/pulibrary/requests/issues/822

Reply:
thanks!